lang/ilias_en.lang

```diff
@@ -14,6 +14,21 @@
 // to the corresponding mode!
 // Language file names refer to ISO 639, see: http://www.oasis-open.org/cover/iso639a.html
 <!-- language file start -->
+changelog#:#delete_old_logs_title#:#Delete old Changelog entries
```

Comment:
Either all words starts in capital letters or only the first one. If it is a button or link phrase or a headline, all words should start with upper case. If it is a sentence or a byline, lower cases are fine.